eos/form-factors/parametric-bsz2015.hh

```diff
@@ -41,15 +115,9 @@ namespace eos
             // use constraint (B.6) in [BSZ2015] to remove A_12(0)
             std::array<UsedParameter, 2> _a_A12, _a_T2;
 
-            const double _mB, _mB2, _mV, _mV2, _kin_factor;
-            const double _tau_p, _tau_0;
-            const double _z_0;
+            const BSZ2015FormFactorTraits<Process_, PToV> _traits;
 
-            static double _calc_tau_0(const double & m_B, const double & m_V);
-
-            complex<double> _calc_z(const complex<double> & s) const;
-
-            double _calc_z(const double & s) const;
+            const double _mB, _mB2, _mV, _mV2, _kin_factor;
```

Comment:
Aren't these handled by ``_traits`` now?

Reply:
They are, but I define them here for simplicity and construct them using the traits. It is just to avoid `this->_traits.mB`...

Reply:
Hm, that raises red flags for me. ``_mB`` and friends are double values, while the traits use ``Parameter`` objects. This can so easily go wrong!

Reply:
I agree, let me change that